new.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: php
```

Comment:
Why do we need this ?

Reply:
We don't.  It must have gotten swept up in my Git commit locally by accident.